src/Components/GrowthChart/GrowthChartOptions/GrowthChartAnnotations.tsx

```diff
@@ -0,0 +1,39 @@
+import i18n from '@dhis2/d2-i18n';
+
+export interface AnnotationLabelType {
+    display: boolean;
+    content?: () => string;
+    position?: 'top' | 'bottom' | 'center' | 'start' | 'end';
+    yAdjust?: number;
+}
+
+export const GrowthChartAnnotations = (xAxisValues: number[], timeUnit: string) => {
+    if (timeUnit === 'Months') {
+        const firstXValue = xAxisValues[0];
+
+        const annotations = xAxisValues
+            .filter((label) => label % 12 === 0)
+            .map((label) => ({
+                display: true,
+                type: 'line',
+                scaleID: 'x',
+                borderWidth: 1.2,
+                value: label - firstXValue,
+                label: {
+                    display: true,
+                    content: () => {
+                        const value = label / 12;
+                        return `${value} ${value === 1 ? i18n.t('Year') : i18n.t('Years')}`;
+                    },
+                    position: 'end',
+                    yAdjust: 10,
+                },
+
+            }));
+        annotations.pop();
```

Comment:
I chose to pop first and last, because they know the "to and from". For example they pick 0 to 2 years, they do not need annotations for 0 and 2, only for 1 year